README.md

```diff
@@ -22,6 +22,10 @@ juju relate keepalived:lb-sink kubeapi-load-balancer:website
 juju relate keepalived:loadbalancer kubernetes-master:loadbalancer
 juju relate keepalived:website kubernetes-worker:kube-api-endpoint
 
+# this relation is also needed as usual so that the actual
+# load-balancer knows about backend endpoints
+juju relate kubernetes-master:kube-api-endpoint kubeapi-load-balancer:apiserver
+
```

Comment:
I'm not sure this note is necessary. The assumption is that you're executing these commands on a CDK model, and therefore this relation would already exist.

Reply:
This is more of a hint for somebody who already has a single kubeapi-loadbalancer and needs to understand which relations to keep when adding keepalived. It was not apparent to me at first - I'd almost forgotten it once.

Reply:
The worker <=> load-balancer http interface-based relation is not made in the new setup. Instead, workers are only related to keepalived. Workers go over all http relations and pick a random hostname out of all lb hostnames (look for random.choice). To avoid that we only need to expose a VIP or a VIP hostname without additional lb endpoints.

Reply:
Another side-effect of using keepalived I noticed in this setting is that ~/config copied from the master node contains an IP address of one of the load-balancer units, not a VIP or VIP hostname.

Worker node ~/.kube/config contains a VIP or VIP hostname on the other hand.

Reply:
Okay, so we need to add this to the instructions?:
```
juju remove-relation kubernetes-worker:kube-api-endpoint kubeapi-load-balancer:website
```

Reply:
Regarding the master config, I'll open a separate bug for that on juju-solutions/bundle-canonical-kubernetes.

Reply:
On removing the relation: yes.

Reply:
Okay, I'll merge and add that separately, thanks.